Lib/test/test_builtin.py

```diff
@@ -1902,6 +1903,13 @@ def __getitem__(self, index):
         self.assertEqual(sum(xs), complex(sum(z.real for z in xs),
                                           sum(z.imag for z in xs)))
 
+        # test that sum() of complex and real numbers doesn't
+        # smash sign of imaginary 0
+        self.assertComplexesAreIdentical(sum([complex(1, -0.0), 1]),
```

Comment:
Does it also work for `sum([1, complex(1, -0.0)])`?

Reply:
```pycon
>>> sum([complex(1, -0.0), 1])
(2-0j)
>>> sum([1, complex(1, -0.0)])
(2-0j)
```
Addition is still commutative, as before.  This input handled just by the PyNumber_Add() in less generic fast summation block (for long's, not for complexes).  I was thinking about adding swapped versions, but probably it doesn't belongs here: the test_complex.py has such tests.

Do you think I should add them, despite some redundancy?